easybuild/easyconfigs/m/MUSCLE3/MUSCLE3-0.7.2-intel-2023b.eb

```diff
@@ -0,0 +1,105 @@
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'MUSCLE3'
+version = '0.7.2'
+
+homepage = 'https://muscle3.readthedocs.io/en/latest/index.html'
+
+description = """MUSCLE3 allows connecting multiple simulation models
+ together into a multiscale simulation. Simulation models can be as
+ simple as a single Python file, or as complex as a combination of
+ multiple separate simulation codes written in C++ or Fortran, and
+ running on an HPC machine."""
+
+toolchain = {'name': 'intel', 'version': '2023b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/multiscale/%(namelower)s/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['9596899f4a576d4ca80cc54c4cd458aac25ef0038a97843c76cec60a7105f779']
+
+builddependencies = [
+    ('googletest', '1.14.0'),
+]
+
+dependencies = [
+    ('msgpack-c', '6.0.0'),
+    ('zlib', '1.2.13'),
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.12'),
+    ('QCG-PilotJob', '0.13.1'),
+    # ('ruamel.yaml', '0.17.40'),
+    ('PyYAML', '6.0.1'),
+    ('matplotlib', '3.8.2'),
```

Comment:
Hi @SimonPinches,
the intel recipe is failing: the dependencies you are using here are from the foss toolchain and not from the intel one. Our framework cannot find them in the intel toolchain diagram and fail to load them

Reply:
I've added the missing dependencies.